index.js

```diff
@@ -157,6 +157,8 @@ class K8sVMExecutor extends Executor {
         this.highMemory = hoek.reach(options, 'kubernetes.resources.memory.high', { default: 12 });
         this.lowMemory = hoek.reach(options, 'kubernetes.resources.memory.low', { default: 2 });
         this.microMemory = hoek.reach(options, 'kubernetes.resources.memory.micro', { default: 1 });
+        this.diskLabel = hoek.reach(options, 'kubernetes.labels.disk',
+            { default: 'screwdriver.cd/disk' });
```

Comment:
same concern about setting the default label as my comment in https://github.com/screwdriver-cd/queue-worker/pull/97#pullrequestreview-154848309.